R/zoom.R

```diff
@@ -0,0 +1,22 @@
+cdm_zoom_to_tbl <- function(dm, table) {
+  # for now only one table can be zoomed on
+  zoom <- as_string(ensym(table))
+  check_correct_input(dm, zoom)
+  zoom_tbl <- tbl(dm, zoom)
```

Comment:
Do we accept filtered dm-s? What happens if the dm is already zoomed?

Reply:
1. filtered dm-s: this is a tricky question, since it very much depends on what the user is planning to do with the zoomed table. I do think there are plenty scenarios, where things could take unintended turns, if we accept filtered dm-s and the user isn't aware that the original (unfiltered) table is used for the zoom. I think I would go for a warning. What do you think?
2. Currently, the originally zoomed table will be just dropped and forgotten, if another table is zoomed on. This would be a more than annoying behaviour, if that happened with a carefully manipulated zoomed table. So maybe let's only allow to zoom on a table if the `dm` is unzoomed?